truss/templates/trtllm/packages/build_engine_utils.py

```diff
@@ -0,0 +1,35 @@
+from pathlib import Path
+
+from schema import EngineBuildArgs
+
+
+def build_engine_from_config_args(
+    engine_build_args: EngineBuildArgs,
+    dst: Path,
+):
+    import sys
+
+    sys.path.append("/app/baseten")
+
+    import os
+    import shutil
+
+    # NOTE: These are provided by the underlying base image
+    from build_engine import Engine, build_engine
+    from trtllm_utils import docker_tag_aware_file_cache
+
+    engine = Engine(**engine_build_args.model_dump())
+
+    with docker_tag_aware_file_cache("/root/.cache/trtllm"):
+        built_engine = build_engine(engine, download_remote=True)
```

Comment:
Let's make a TODO to change `build_engine` to be able to build the engine directly to the destination path to avoid copying over these files